torch/_torch_docs.py

```diff
@@ -6268,6 +6268,53 @@ def merge_dicts(*dicts):
     tensor([  2.,   4.,   8.,  16.])
 """.format(**common_args))
 
+add_docstr(torch.float_power,
+           r"""
+float_power(input, exponent, *, out=None) -> Tensor
+
+Raises :attr:`input` to the power of :attr:`exponent`, elementwise, in double precision. 
+If neither input is complex returns a ``torch.float64`` tensor, 
+and if one or more inputs is complex returns a ``torch.complex128`` tensor.
+
+:attr:`exponent` can be either a single number, or a `Tensor`
+with the same number of elements as :attr:`input`.
+
+.. math::
```

Comment:
This mathematical portion is also correct but can be removed. The description above it is already very clear.

Reply:
Addressed!